appsscript.json

```diff
@@ -16,8 +19,6 @@
   "oauthScopes": [
     "https://www.googleapis.com/auth/script.container.ui",
     "https://www.googleapis.com/auth/script.external_request",
-    "https://www.googleapis.com/auth/spreadsheets",
```

Comment:
My reading of the Google documentation regarding these scopes is that we need them for this to be considered a "Sheets" project, but we'll try them without.